auth_proxy/application.py

```diff
@@ -16,6 +16,7 @@
 
 app.config['API_SERVER'] = os.getenv('API_SERVER')
 app.config['API_SERVER_NAME'] = os.getenv('API_SERVER_NAME')
+app.config['ENABLE_UNSECURE_FHIR'] = os.getenv('ENABLE_UNSECURE_FHIR', '') == 'True'
```

Comment:
`os.getenv('ENABLE_UNSECURE_FHIR')` should work just as well, no (I mean, without the `, ''`)?

Reply:
Correct, artifact of me futzing with bools.